conda/bootstrap-conda.sh

```diff
@@ -37,11 +37,18 @@ else
     fi
     ## specify Miniconda release (e.g., MINICONDA_VERSION='4.0.5')
     if [[ -z "${MINICONDA_VERSION}" ]]; then
-        echo "MINICONDA_VERSION not set, setting ..."
-        # Pin to 4.2.12 by default until Spark default is 2.2.0.
-        # https://issues.apache.org/jira/browse/SPARK-19019
+      # Pin to 4.2.12 by default until Spark default is 2.2.0, then use latest
+      # https://issues.apache.org/jira/browse/SPARK-19019
+      MIN_SPARK_VERSION="2.2.0"
```

Comment:
Let's move this constant to the top

Reply:
Done: moved and renamed with more specific name